types/react-sortable-tree/index.d.ts

```diff
@@ -57,12 +57,22 @@ export interface ExtendedNodeData extends NodeData {
 export interface OnVisibilityToggleData extends FullTree, TreeNode {
     expanded: boolean;
 }
+
 export interface PreviousAndNextLocation {
+    prevTreeIndex: number;
+    prevPath: number[];
+    prevParent: TreeItem | null;
+    nextTreeIndex: number;
+    nextPath: number[];
+    nextParent: TreeItem | null;
+}
+
+export interface OnMovePreviousAndNextLocation {
```

Comment:
Both interfaces have duplicate properties. What about `OnMovePreviousAndNextLocation` extends `PreviousAndNextLocation`?

Reply:
@jzoric thanks for your input. I cannot extend from `PreviousAndNextLocation` otherwise `prevParent` and `nextParent` will be available in `OnMovePreviousAndNextLocation`, unless there is a way to partially extend an interface which I don't know about.

I could create a common interface and extend them both
```js
export interface PreviousAndNextLocation {
    prevTreeIndex: number;
    prevPath: number[];
    nextTreeIndex: number;
    nextPath: number[];
}

export interface onDragPreviousAndNextLocation extends PreviousAndNextLocation {
    prevParent: TreeItem | null;
    nextParent: TreeItem | null;
}

export interface OnMovePreviousAndNextLocation extends PreviousAndNextLocation {
    nextParentNode: TreeItem | null;
}
```

Reply:
Sounds good. In that case, do we need to export `PreviousAndNextLocation`? Only to export 
`OnDragPreviousAndNextLocation ` and `OnMovePreviousAndNextLocation` interfaces.

Reply:
You are right, I won't export `PreviousAndNextLocation`. I'll update the pr soon.